docs/source/foundations.rst

```diff
@@ -99,10 +99,12 @@ Server Side
            data += 1
            await comm.write(data)
 
-   s = Server({'add': add, 'stream_data': stream_data})
-   s.listen('tcp://:8888')   # listen on TCP port 8888
```

Comment:
`s.listen(` was changed to be an  `async def` so this didn't work anymore: https://github.com/dask/distributed/blob/0b5d6eb26c202825bbd2140452c51262406e8eae/distributed/core.py#L444-L461